wp-content/themes/pub/wporg-learn-2020/template-parts/component-post-card.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * The template part for display a post card
+ *
+ * @package WordPressdotorg\Theme;
+ */
+
+namespace WordPressdotorg\Theme;
+
+$args = wp_parse_args( $args );
+$post_type_name = get_post_type();
+$post_type_info = get_post_type_object( $post_type_name );
+
+?>
+<article class="post-card">
+	<?php if ( ! in_array( $post_type_name, array( 'page', 'attachment', 'post' ) ) ) : ?>
+	<div class="post-card_tag post-card_tag--<?php echo esc_attr( $post_type_name ); ?>">
+		<?php echo esc_attr( $post_type_info->labels->singular_name ); ?>
+	</div>
+	<?php endif; ?>
+	<div class="entry">
+		<header class="entry-header">
+			<h3 class="entry-title">
```

Comment:
These should be `h2` for heading hierarchy (since the search title is an h1).

Reply:
it looks like this wasn't changed, is there a reason to keep it `h3`?

Reply:
No, I didn't push back my changes before the end of the day. It's an `h2` now.